pkg/steps/release/release_images.go

```diff
@@ -122,10 +122,7 @@ func (s *releaseImagesTagStep) Inputs(ctx context.Context, dry bool) (api.InputD
 }
 
 func sourceName(config api.ReleaseTagConfiguration) string {
-	if len(config.Name) > 0 {
-		return fmt.Sprintf("%s/%s:${component}", config.Namespace, config.Name)
-	}
-	return fmt.Sprintf("%s/${component}:%s", config.Namespace, config.Tag)
```

Comment:
Didn't you just make a type for `${component}`

Reply:
global constant yes.

Reply:
Fixed